apps/web/.env.example

```diff
@@ -50,7 +50,7 @@ GCP_CLIENT_EMAIL=
 GCP_PRIVATE_KEY=
 CRON_SECRET=
 
-EXTERNAL_API_URL=
```

Comment:
keep the `EXTERNAL_API_URL` as its being used to fetch some data (and simply extend with `VERCEL_URL`)